api/src/glfs-fops.c

```diff
@@ -5681,14 +5681,17 @@ glfs_recall_lease_upcall(struct glfs *fs, struct glfs_upcall *up_arg,
         goto out;
     }
 
-    up_lease_arg = GF_CALLOC(1, sizeof(struct glfs_upcall_lease),
+    up_lease_arg = GF_MALLOC(sizeof(struct glfs_upcall_lease),
                              glfs_mt_upcall_inode_t);
+    if (!up_lease_arg) {
+        errno = ENOMEM;
+        goto out;
+    }
     up_lease_arg->object = object;
+    up_lease_arg->lease_type = recall_lease->lease_type;
 
     GF_VALIDATE_OR_GOTO("glfs_recall_lease", up_lease_arg, out);
```

Comment:
This is useless here. It should be removed.

Reply:
Are you talking about line number 5693 ?